### ISSUE TITLE
System for more settings

### ISSUE THREAD

**Issue by TheColaber** [ScratchAddons/ScratchAddons#2297]
**Is your feature request related to a problem? Please describe.**

Right now, if you want to create a new setting more "more settings", you would have to manually add to the dom and would have a bad interface for controlling what happens when you switch settings or press buttons.

**Describe the solution you'd like**

Create a system to easily add new "more setting", and maybe provide some apis.

**Describe alternatives you've considered**

This current wack.

**Comment by mxmou** [comment]
I don't think this is necessary because the More settings popup isn't changed often.

**Comment by TheColaber** [comment]
> I don't think this is necessary because the More settings popup isn't changed often.

But I don't think it will stay like that forever. For example, #10, https://github.com/ScratchAddons/ScratchAddons/pull/2260#issuecomment-826787092, and others can benefit from more settings. Also, this would help us organize the settings page more.

**Comment by GrahamSH-LLK** [comment]
When making this modal, I specifically asked if I should create something like this, and @WorldLanguages said no.
